client/src/main/java/com/vesoft/nebula/client/storage/data/EdgeTableRow.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.client.storage.data;
+
+import java.util.List;
+
+public class EdgeTableRow extends BaseTableRow {
+
+    public EdgeTableRow(List<Object> values) {
+        super(values);
+    }
+
+    public EdgeTableRow(List<Object> values, String decodeType) {
+        super(values, decodeType);
+    }
+
+    //todo int srcId
+    public String getSrcId() {
```

Comment:
ditto

Reply:
TODO has been flaged， int id will be handled in another pr.